Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_description.md

```diff
@@ -22,4 +22,6 @@
 
 **Close-reason default mapping XDR -> XSOAR**: _Known Issue=Other, Duplicate Incident=Duplicate, False Positive=False Positive, True Positive=Resolved, Other=Other, Auto=Resolved_
 
+**Close Mirrored Cortex XDR Incident**: In case this checkbox is not selected but **Close all related alerts in XDR** is selected, the incident will closed automatically,
```

Comment:
```suggestion
**Close Mirrored Cortex XDR Incident**: In case this checkbox is not selected but **Close all related alerts in XDR** is selected, the incident will be closed automatically,
```